test_move_command_too_few_paths FAILs in downstream testing of dandischema

```
(base) dandi@drogon:/mnt/backup/dandi/tinuous-logs/dandischema/2022/06$ git grep  'test_move_command_too_few_paths FAILED' 2*
28/github/cron/20220628T060247/148a18c/test-dandi-cli.yml/583-failed/7_test-dandi-cli (ubuntu-18.04, 3.8, dandi-devel, master).txt:2022-06-28T06:04:25.6803846Z cli/tests/test_move.py::test_move_command_too_few_paths FAILED
28/github/cron/20220628T060247/148a18c/test-dandi-cli.yml/583-failed/8_test-dandi-cli (ubuntu-18.04, 3.8, dandi-devel, release).txt:2022-06-28T06:04:36.8729715Z cli/tests/test_move.py::test_move_command_too_few_paths FAILED
28/github/cron/20220628T060247/148a18c/test-dandi-cli.yml/583-failed/test-dandi-cli (ubuntu-18.04, 3.8, dandi-devel, master)/12_Run dandi-cli tests.txt:2022-06-28T06:04:25.6803787Z cli/tests/test_move.py::test_move_command_too_few_paths FAILED
28/github/cron/20220628T060247/148a18c/test-dandi-cli.yml/583-failed/test-dandi-cli (ubuntu-18.04, 3.8, dandi-devel, release)/12_Run dandi-cli tests.txt:2022-06-28T06:04:36.8729646Z cli/tests/test_move.py::test_move_command_too_few_paths FAILED
29/github/cron/20220629T060249/148a18c/test-dandi-cli.yml/584-failed/7_test-dandi-cli (ubuntu-18.04, 3.8, dandi-devel, master).txt:2022-06-29T06:04:47.5194769Z cli/tests/test_move.py::test_move_command_too_few_paths FAILED
29/github/cron/20220629T060249/148a18c/test-dandi-cli.yml/584-failed/8_test-dandi-cli (ubuntu-18.04, 3.8, dandi-devel, release).txt:2022-06-29T06:04:36.1638960Z cli/tests/test_move.py::test_move_command_too_few_paths FAILED
29/github/cron/20220629T060249/148a18c/test-dandi-cli.yml/584-failed/test-dandi-cli (ubuntu-18.04, 3.8, dandi-devel, master)/12_Run dandi-cli tests.txt:2022-06-29T06:04:47.5194707Z cli/tests/test_move.py::test_move_command_too_few_paths FAILED
29/github/cron/20220629T060249/148a18c/test-dandi-cli.yml/584-failed/test-dandi-cli (ubuntu-18.04, 3.8, dandi-devel, release)/12_Run dandi-cli tests.txt:2022-06-29T06:04:36.1638878Z cli/tests/test_move.py::test_move_command_too_few_paths FAILED
```

```
2022-06-28T06:14:34.0228583Z
2022-06-28T06:14:34.0231819Z =================================== FAILURES ===================================
2022-06-28T06:14:34.0232703Z _______________________ test_move_command_too_few_paths ________________________
2022-06-28T06:14:34.0233119Z
2022-06-28T06:14:34.0233761Z mocker = <pytest_mock.plugin.MockerFixture object at 0x7f2d333de2b0>
2022-06-28T06:14:34.0234200Z
2022-06-28T06:14:34.0235010Z     def test_move_command_too_few_paths(mocker: MockerFixture) -> None:
2022-06-28T06:14:34.0235615Z         mock_move = mocker.patch("dandi.move.move")
2022-06-28T06:14:34.0236422Z         r = CliRunner().invoke(move, ["foo"], standalone_mode=False)
2022-06-28T06:14:34.0236942Z         assert r.exit_code != 0
2022-06-28T06:14:34.0237717Z >       assert isinstance(r.exception, click.ClickException)
2022-06-28T06:14:34.0238258Z E       AssertionError: assert False
2022-06-28T06:14:34.0239306Z E        +  where False = isinstance(ValueError('At least two paths are required'), <class 'click.exceptions.ClickException'>)
2022-06-28T06:14:34.0240258Z E        +    where ValueError('At least two paths are required') = <Result ValueError('At least two paths are required')>.exception
2022-06-28T06:14:34.0241344Z E        +    and   <class 'click.exceptions.ClickException'> = click.ClickException
2022-06-28T06:14:34.0241806Z
2022-06-28T06:14:34.0242655Z /opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/dandi/cli/tests/test_move.py:89: AssertionError
2022-06-28T06:14:34.0243486Z ------------------------------ Captured log call -------------------------------
2022-06-28T06:14:34.0244368Z DEBUG    dandi:base.py:110 Caught exception At least two paths are required
2022-06-28T06:14:34.0244944Z =============================== warnings summary ===============================
```

--- a/dandi/cli/tests/test_move.py
+++ b/dandi/cli/tests/test_move.py
@@ -86,6 +86,8 @@ def test_move_command_too_few_paths(mocker: MockerFixture) -> None:
     mock_move = mocker.patch("dandi.move.move")
     r = CliRunner().invoke(move, ["foo"], standalone_mode=False)
     assert r.exit_code != 0
-    assert isinstance(r.exception, click.ClickException)
+    # This is a ClickException when map_to_click_exceptions is in effect and a
+    # ValueError when it's not (which happens when DANDI_DEVEL is set).
+    assert isinstance(r.exception, (click.ClickException, ValueError))
     assert str(r.exception) == "At least two paths are required"
     mock_move.assert_not_called()

